### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Vuurvos1** [davidvanr21/functional-programming#1]
I would recommend getting started fast with working with the parking data and setting up research questions so you can search the datasets more efficiently for the information you need.

### Research questions
Think about interesting topics to write an article about, where can I park the cheapest, what kind of parking spots are where (
disabled parking space and spots to charge electric cars) and so on...

### Add one or more research question to the readme
- What data do you need and from which datasets
- Which assumptions do you have by the research question

### Other things to put in your readme:
- Some more info about the project, what can I expect in the repo?
- How to install and use the project

**Comment by gijslaarman** [comment]
Yes what he's saying. I'm seeing you have some small catching up to do, but you're still fine if you start now. You only need to come up with a research question. If you need any help PM me 👉
